themes/bootstrap3/templates/myresearch/menu.phtml

```diff
@@ -8,102 +8,92 @@
 <h3><?=$this->transEsc('Your Account')?></h3>
 <div class="myresearch-menu account-menu">
   <?php if ($this->userlist()->getMode() !== 'disabled'): ?>
-    <a href="<?=$this->url('myresearch-favorites')?>"<?=$this->active == 'favorites' ? ' class="active"' : ''?>>
-      <?=$this->icon('user-favorites') ?>
-      <?=$this->transEsc('Favorites')?>
+    <a href="<?=$this->url('myresearch-favorites')?>" class="icon-link <?=$this->active == 'favorites' ? ' active' : ''?>">
+      <?=$this->icon('user-favorites', 'icon-link__icon') ?>
+      <span class="icon-link__label"><?=$this->transEsc('Favorites')?></span>
     </a>
   <?php endif; ?>
   <?php if ($ilsOnline && $this->ils()->checkCapability('getMyTransactions', $capabilityParams)): ?>
-    <a href="<?=$this->url('myresearch-checkedout')?>" class="flex checkedout<?=$this->active == 'checkedout' ? ' active' : ''?>">
-      <span class="flex-col">
-        <?=$this->icon('user-checked-out') ?>
-        <?=$this->transEsc('Checked Out Items')?>
-      </span>
-      <span class="checkedout-status status hidden"><?=$this->icon('spinner') ?></span>
+    <a href="<?=$this->url('myresearch-checkedout')?>" class="checkedout icon-link<?=$this->active == 'checkedout' ? ' active' : ''?>">
+      <?=$this->icon('user-checked-out', 'icon-link__icon') ?>
+      <span class="icon-link__label"><?=$this->transEsc('Checked Out Items')?></span>
+      <span class="checkedout-status status icon-link__icon hidden"><?=$this->icon('spinner') ?></span>
```

Comment:
Something weird is going on with wrapping when there are a lot of statuses. See, for example:

![image](https://user-images.githubusercontent.com/309069/197262400-8a998c6e-41b9-40f3-b9b3-d034d2cd3bb7.png)

(compared to the current dev branch:
![image](https://user-images.githubusercontent.com/309069/197262466-7ae36774-a74e-4c3c-81c7-15dac38b6423.png)
)

This is a little bit hard to reproduce. Here's my advice:

1.) Turn on the Demo driver
2.) Go to the profile page and log in (make up whatever credentials you like) to activate patron functionality
3.) Keep logging in and out until you get three boxes next to Checked Out Items. Since things are re-randomized every time, it will take several attempts. To improve the odds, you can bump up the transaction count by modifying the $trans variable at the top of getRandomTransactionList in the demo driver.

I don't think it matters whether or not you have historic transactions turned on, but if you need them for some reason, they're off by default but can be activated in Demo.ini's [TransactionHistory] section.

Reply:
Another easy way to test this is to edit the session storage in the browser to something like `{ "ok": 10, "warn": 1, "overdue": 5 }`.

Reply:
Yes, that's much easier -- thanks!